domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/topic/domain/TopicPersistence.kt

```diff
@@ -0,0 +1,6 @@
+package com.krzykrucz.elesson.currentlesson.domain.topic.domain
+
+import arrow.fx.IO
+
+typealias CountFinishLessons = () -> IO<FinishedLessonsCount>
```

Comment:
shouldn't it be named `CountFinishedLessons` ?

Reply:
Yup, done